scripts/collector/postgres/sql/op_collect.sql

```diff
@@ -0,0 +1,20 @@
+\i sql/init.sql
+\i sql/applications.sql
+--\i sql/awsextensiondependency.sql
+--\i sql/awsoracleexists.sql
+--\i sql/awsoracleversion.sql
```

Comment:
What's the reason for disabling these right now?

Reply:
I'll have to check.